src/plugins/vis_type_timeseries/server/routes/vis.ts

```diff
@@ -43,12 +44,6 @@ export const visDataRoutes = (
         visPayloadSchema.validate(request.body);
       } catch (error) {
         logFailedValidation();
-        const savedObjectId =
-          (typeof request.body === 'object' && (request.body as any).savedObjectId) ||
-          'unavailable';
-        framework.logger.warn(
```

Comment:
why you decided to remove schema validation warning message?

Reply:
the `savedObjectId` param is not used in visualization

Reply:
?? how it's related to that message? 
Please try the following case: 
1) go to `src/plugins/vis_type_timeseries/common/vis_schema.ts` and break the schema
2) refresh TSVB visualization and see console 

I expect to see warning message that schema is not valid. 
![image](https://user-images.githubusercontent.com/20072247/99676514-a9a54980-2a89-11eb-9a34-559ecebfad5b.png)

Reply:
ok I see now, thanks!
Returned back